src/app/components/modals/courseInfoModal/LabelSelectionModal.tsx

```diff
@@ -145,12 +148,14 @@ function LabelListItem({
   handleToggle,
   isChecked,
   onLabelChanged,
+  showCheck,
 }: {
   label: Label;
   index: number;
   handleToggle: (label: Label) => void;
   isChecked: boolean;
   onLabelChanged: (label: Label) => void;
+  showCheck: boolean;
```

Comment:
Not sure what the point of the `showCheck` prop is. I think this worked fine before.